include/gunrock/algorithms/bc.hxx

```diff
@@ -115,6 +125,11 @@ struct enactor_t : gunrock::enactor_t<problem_t> {
 
     auto bc_values = P->result.bc_values;
 
+    auto edges_visited = P->result.edges_visited;
+    auto search_depth = P->result.search_depth;
+
+    auto performance = P->param.performance;
```

Comment:
This might be a tedious change (sorry!) but I prefer `collect_metrics` or something like that instead of `performance`.